airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, job_name, job_definition, job_queue, overrides, array_propert
         self.job_definition = job_definition
         self.job_queue = job_queue
         self.overrides = overrides
-        self.array_properties = array_properties
+        self.array_properties = array_properties if array_properties is not None else {}
```

Comment:
```suggestion
        self.array_properties = array_properties or {}
```
Why not just like that?

Reply:
```python
>>> a = None or {}
>>> a
{}
```

Reply:
lovely